wagtail/admin/templates/wagtailadmin/notifications/task_state_approved_subject.txt

```diff
@@ -1,3 +1,11 @@
-{% load i18n %}
+{% load wagtailadmin_tags i18n %}
 
-{% blocktrans trimmed with title=page.get_admin_display_title|safe task=task.name|safe %}The page "{{ title }}" has been approved in "{{ task }}".{% endblocktrans %}
+{% if page %}
+    {% blocktrans trimmed with title=page.get_admin_display_title|safe task=task.name|safe %}The page "{{ title }}" has been approved in "{{ task }}".{% endblocktrans %}
+{% else %}
+    {% latest_str object as title %}
+    {% blocktrans trimmed with title=title|safe task=task.name|safe model_name=model_name|safe asvar message %}
+        The {{ model_name }} "{{ title }}" has been approved in "{{ task }}".
+    {% endblocktrans %}
+    {{ message|capfirst|safe }}
```

Comment:
And here, but I'll stop noting all the templates now - just a generic question about the use of `safe` for all.